nginx/nginx.conf

```diff
@@ -13,6 +13,13 @@ http {
     keepalive_timeout     3600;
     proxy_read_timeout    3600;
     client_max_body_size  32M;
+    server_tokens         off;
+
+    map $http_accept $error_extension {
+        # Used by conf.d/error_pages.conf
+        default html;
+        ~application/json json;
+    }
```

Comment:
very elegant fix to detect `html` vs `json` based on client's requested `http_accept`!